simp/pupmod-simp-mozilla
# (SIMP-10204) Ensure testing with CentOS 8.4

- Add option to package resource to  enable ol7_optional_latest
  repo for OEL 7.
- Test Updates
  - update the EL8 nodes in the acceptance tests
    to run with generic/centos8 to make sure they using
    CentOS 8.4
  - update oel7 node to use generic bx instead of onyxpoint
    and add the oel8 node.

SIMP-10204 #comment mozilla update
SIMP-10230 #close
